docs/CONTRIBUTING.md

```diff
@@ -2,4 +2,5 @@
 
 * We use the [PEP8](https://peps.python.org/pep-0008/) coding standard
 * OME-Zarr format extension should be `ome.zarr`, as shown in the NGFF documentation ([URL](https://ngff.openmicroscopy.org/0.4/index.html#bf2raw-layout))
+* All resolutions and dimensions should be given in micron (µm)
```

Comment:
I would remove this. All scripts should specify the units in which the data is expected.

Reply:
Modified this in [cfbc942](https://github.com/linum-uqam/linumpy/pull/32/commits/cfbc9425c1d77616de9218c1da9b92f975dbcf9f)